sencha-workspace/packages/slate-cbl/src/view/demonstrations/SkillList.js

```diff
@@ -116,16 +125,114 @@ Ext.define('Slate.cbl.view.demonstrations.SkillList', {
+                                                            <i class="fa fa-link skill-list-link-icon"></i>
+                                                            <div class="skill-list-link-label">{Demonstration.ArtifactURL}</div>
+                                                        </a>
+                                                    </li>
+                                                </ul>
+                                            </div>
+                                        </tpl>
+                                        <tpl if="Demonstration.StudentTask.Task.Attachments && Demonstration.StudentTask.Task.Attachments.length">
+                                            <div class="skill-list-detail-group">
+                                                <h5 class="skill-list-detail-subheading">Task Attachments</h5>
+                                                <ul class="skill-list-links">
+                                                    <tpl for="Demonstration.StudentTask.Task.Attachments">
+                                                        <li class="skill-list-link-item">
+                                                            <a href="{URL}" target="_blank" class="skill-list-link">
+                                                                <i class="fa fa-link skill-list-link-icon"></i>
+                                                                <div class="skill-list-link-label">{[this.linkText(values)]}</div>
+                                                            </a>
+                                                        </li>
+                                                    </tpl>
+                                                </ul>
+                                          </div>
+                                        </tpl>
+                                        <tpl if="this.hasActiveAttachments(values.Demonstration.StudentTask.Attachments) == true">
+                                            <div class="skill-list-detail-group">
+                                                <h5 class="skill-list-detail-subheading">Task Submissions</h5>
+                                                <ul class="skill-list-links">
+                                                    <tpl for="Demonstration.StudentTask.Attachments">
+                                                    <tpl if="Status != 'removed'">
+                                                            <li class="skill-list-link-item">
+                                                                <a href="{URL}" target="_blank" class="skill-list-link">
+                                                                    <i class="fa fa-link skill-list-link-icon"></i>
+                                                                    <div class="skill-list-link-label">{[this.linkText(values)]}</div>
```

Comment:
```suggestion
                                                                    <div class="skill-list-link-label">{[fm.htmlEncode(this.linkText(values))]}</div>
```

I _think_ this will work in this context, xtemplate javascript has access to an `fm` alias to `Ext.util.Format`

Reply:
resolved 1ffe9c6d165268ee08b8824413a0a8aa06eace14